src/ripple/app/main/Main.cpp

```diff
@@ -181,6 +181,39 @@ void printHelp (const po::options_description& desc)
 
 //------------------------------------------------------------------------------
 
+/* simple unit test selector that allows a comma separated list
+ * of selectors
+ */
+class multi_selector
+{
+private:
+    std::vector <beast::unit_test::selector> selectors_;
+public:
+    explicit
+    multi_selector(std::string const& patterns = "")
+    {
+        std::vector<std::string> v;
+        boost::split (v, patterns, boost::algorithm::is_any_of (","));
+        selectors_.reserve(v.size());
+        std::for_each(v.begin(), v.end(),
+            [this](std::string const& s)
+            {
```

Comment:
I expected to see code that would strip whitespace off the ends - either here or in the selector constructor. Do we need to do that? I'd probably strip it off here we need to.

Reply:
I agree we should trim here - will add